### ISSUE TITLE
Change Anchors Size - Really small object

### ISSUE THREAD

**Issue by VincentDuf** [OlafenwaMoses/ImageAI#562]
Hey,
I'm trying to train a custom model to detect aphids on plants, the aphids are really small and one advice I've been given is to reduce the size of the anchors.

Is there a way to do it properly ? 

Thank you

**Comment by AbimbolaOO** [comment]
Hello @VincentDuf,

ImageAI doesn't provide a way to do so.

**Comment by alexvmt** [comment]
I'm also interested in reducing the default anchor size since I'm trying to detect small objects. Is there no way to manually change the default anchor size? Has anyone tried? @VincentDuf did you figure out a solution for your problem?

**Comment by alexvmt** [comment]
I think I found the script that generates custom anchors: https://github.com/OlafenwaMoses/ImageAI/blob/master/imageai/Detection/Custom/gen_anchors.py

After skimming through it briefly it appears to me that anchors are generated depending on the bounding box coordinates one provides in the annotation files.

**Comment by VincentDuf** [comment]
Yep, I also found this script but I didn't get how they calculated their anchors so I asked but as you can see above it seems not possible to change their size (I suppose this would make crash the program)

**Comment by OlafenwaMoses** [comment]
Anchor sizes are calculated based on the box sizes of your annotations, just as @alexvmt said.
